### DISCUSSION TITLE
为何使用 jsx 而不是 <template>

### DISCUSSION THREAD

**Discussion by trcat** [tusen-ai/naive-ui#6509] [Q&A] [answered]
vue3 会更推荐我们用SFC的写法来写组件，根据 vue 的描述，`<template>` 在编译之后还有更新逻辑的优化，但是看到 naive ui 中选择使用 jsx，为何选择 jsx 而不是SFC，是否是有特别的考虑

**Comment by jahnli** [comment, marked as answer]
I also mentioned this problem at the time, and later found that `jsx` was more convenient in some scenes.

https://github.com/tusen-ai/naive-ui/issues/216#issuecomment-867565203

**Reply by trcat** [reply]
oh thanks，that is what I'm look ing for～
